root/types.js

```diff
@@ -214,12 +214,12 @@ type CatalystStashT = {
   +collaborative_collections?: $ReadOnlyArray<CollectionT>,
   +commons_image?: CommonsImageT | null,
   +containment?: {
-    [number]: ?1,
+    [collection_id: number]: ?1,
```

Comment:
`collection_id` is used here https://github.com/metabrainz/musicbrainz-server/blob/master/root/static/scripts/common/utility/storage.js#L62